superset/reports/commands/base.py

```diff
@@ -60,4 +61,4 @@ def validate_chart_dashboard(
                 exceptions.append(DashboardNotFoundValidationError())
             self._properties["dashboard"] = dashboard
         elif not update:
-            exceptions.append(ReportScheduleChartOrDashboardValidationError())
+            exceptions.append(ChartNotSavedValidationError())
```

Comment:
is it possible that this could be a dashboard?

Reply:
From my understanding dashboards would always be saved and have a record in the metastore (we allow for empty dashboards), but for a more defensive approach i'll make a specific condition for both dashboards and charts just in case.